File: BTMS.Admin/ClientApp/src/app/models/viewmodels/seat-view-model.ts
import { BookingStatus } from "../data/enums";

export interface SeatViewModel {
   seatSerial?:number|undefined
   seatNumber?:string|undefined;
   status?:BookingStatus|undefined;
   bus?:string|undefined;
   busRoute?:string|undefined;
   fare?:number|undefined;
   scheduleTime?:Date|undefined;
}

File: BTMS.Data/Models/ViewModels/SeatViewModel.cs
using BTMS.DataLib.Models;
using System;

namespace BTMS.Data.Models.ViewModels
{
    public class SeatViewModel
    {
        
        public int SeatSerial { get; set; }
        public string SeatNumber { get; set; }
        public BookingStatus Status { get; set; }
        public string Bus { get; set; }
        public string BusRoute { get; set; }
        public decimal Fare { get; set; }
        public DateTime ScheduleTime { get; set; }

    }
}
